sunpy/visualization/colormaps/color_tables.py

```diff
@@ -312,51 +369,10 @@ def std_gamma_2():
 
 
 def euvi_color_table(wavelength: u.angstrom):
-
     try:
         return cmap_from_rgb_file(f'EUVI {str(wavelength)}', f'euvi_{int(wavelength.value)}.csv')
     except OSError:
         raise ValueError(
             "Invalid EUVI wavelength. Valid values are "
             "171, 195, 284, 304."
         )
-
-
-def cmap_from_rgb_file(name, fname):
```

Comment:
I moved this first just because they get called by other functions and I can't handle them being at the end of the file.